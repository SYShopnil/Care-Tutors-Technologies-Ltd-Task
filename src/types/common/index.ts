export enum EDataTestId {
  SHeaderMain = "SHeaderMain",
  CToDoCreateUpdate = "CToDoCreateUpdate",
  CTodoItem = "CTodoItem",
  CInputForm = "CInputForm",
  CToDoList = "CToDoList",
  CButton = "CButton",
}
