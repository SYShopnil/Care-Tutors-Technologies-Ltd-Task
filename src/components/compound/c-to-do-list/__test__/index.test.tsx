import { render, screen } from "@testing-library/react";
import { EDataTestId } from "@src/types/common";
import CStoreProvider from "../../c-store-provider";

import CTodoList from "..";

describe("When To Do List Rendered", () => {
  it("Expect render successfully", async () => {
    render(
      <CStoreProvider>
        <CTodoList />
      </CStoreProvider>
    );
    const myElement = screen.getByRole(EDataTestId.CToDoList);
    expect(myElement).toMatchSnapshot();
  });
});
