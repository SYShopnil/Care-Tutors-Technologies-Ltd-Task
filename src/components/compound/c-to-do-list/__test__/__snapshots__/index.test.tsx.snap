// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When To Do List Rendered Expect render successfully 1`] = `
<div
  role="CToDoList"
>
  <div>
    <div
      class="flex justify-center items-center h-[50vh]"
    >
      <p
        class="text-center font-extrabold text-2xl text-[#7F4D4F]"
      >
        No ToDo Found!!
      </p>
    </div>
  </div>
</div>
`;
